src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportDestinationClient/IExportDestinationClient.cs

```diff
@@ -29,6 +29,16 @@ public interface IExportDestinationClient
         /// <returns>A <see cref="Task"/> representing connection operation.</returns>
         Task ConnectAsync(string connectionSettings, CancellationToken cancellationToken, string containerId = null);
 
+        /// <summary>
```

Comment:
Have you considered maybe delegate the "authentication" part and glue them together in IoC container?

Reply:
Removed the `ConnectWithAccessTokenAsync` method